proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -111,6 +118,8 @@ service AdminService {
   rpc RunServerUniquenessBackfillTask(RunServerUniquenessBackfillTaskRequest) returns (RunServerUniquenessBackfillTaskResponse) {
     option idempotency_level = IDEMPOTENT;
   }
+  // Update review flow grace period policy for the server.
+  rpc UpdateGracePeriodPolicy(UpdateGracePeriodPolicyRequest) returns (UpdateGracePeriodPolicyResponse);
```

Comment:
Will the front end need to render an existing/currently set grace period? Do we need a `Get` for that?

Reply:
added in https://github.com/bufbuild/buf/pull/2369/commits/b2df0de1e196a4fbb6b95c0f51b446c2c0e1e2b2, thanks!